build/userale-2.4.0.min.js

```diff
@@ -15,4 +15,783 @@
+!(function (e, t) {
```

Comment:
Something is not correct with your build. This file should be single line, if you run `npm install` followedd by `npm build` is this what you get?

Reply:
hey, first when i saw this code i thought it was a mistake so i formatted it using prettier to have the proper indentation, anyways its not a problem, its the same code. I will revert the changes